packages/openneuro-components/src/dataset/DraftDatasetPageExample.tsx

```diff
@@ -0,0 +1,296 @@
+              isMarkdown={true}
+              className="dmb-list"
+            />
+          </>
+        )}
+        renderBrainLifeButton={() => (
+          <BrainLifeButton
+            datasetId={datasetId}
+            onBrainlife={dataset.onBrainlife}
+          />
+        )}
+        renderValidationBlock={() => <ValidationBlock />}
+        renderCloneDropdown={() => (
+          <CloneDropdown datasetId={datasetId} gitHash={dataset.draft.head} />
+        )}
+        renderHeader={() => (
+          <DatasetHeader
+            pageHeading={description.Name}
+            modality={summary.modalities[0]}
+          />
+        )}
+        renderAlert={() => <>{isPublic && <DatasetAlert />}</>}
+        renderHeaderMeta={() => (
+          <DatasetHeaderMeta
+            size={summary.size}
+            totalFiles={summary.totalFiles}
+            datasetId={datasetId}
+          />
+        )}
+        renderToolButtons={() => (
+          <>
+            <Tooltip tooltip="Publish the dataset publicly" flow="up">
+              <Button
+                className="dataset-tool"
+                onClick={() => goToToolPath(history, rootPath, 'publish')}
+                label="Publish"
+                icon="fa fa-globe"
+              />
+            </Tooltip>
+            <Tooltip tooltip="Share this dataset with collaborators" flow="up">
+              <Button
+                className="dataset-tool"
+                onClick={() => goToToolPath(history, rootPath, 'share')}
+                label="Share"
+                icon="fa fa-user"
+              />
+            </Tooltip>
+
+            <Tooltip tooltip="Create a new version of the dataset" flow="up">
+              <Button
+                className="dataset-tool"
+                onClick={() => goToToolPath(history, rootPath, 'snapshot')}
+                label="Snapshot"
+                icon="fa fa-camera"
+              />
+            </Tooltip>
+            <span>
+              <Button
+                className="dataset-tool"
+                onClick={() => goToToolPath(history, rootPath, 'download')}
+                label="Download"
+                icon="fa fa-download"
+              />
+            </span>
+            <Tooltip
+              wrapText={true}
+              tooltip="A form to describe your dataset (helps colleagues discover your dataset)"
+              flow="up">
+              <Button
+                className="dataset-tool"
+                onClick={() => goToToolPath(history, rootPath, 'metadata')}
+                label="Metadata"
+                icon="fas fa-file-code"
+              />
+            </Tooltip>
+            <Tooltip tooltip="Remove your dataset from OpenNeuro" flow="up">
+              <Button
+                className="dataset-tool"
+                onClick={() => goToToolPath(history, rootPath, 'delete')}
+                label="Delete"
+                icon="fas fa-trash"
+              />
+            </Tooltip>
```

Comment:
I think she was saying that they would be better as Buttons or Links instead of warnButtons, I think. It would be easy enough to change them to `<Link to={rootpath+'/delete'} />` though

Reply:
changed to links